extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -43,6 +43,18 @@ public class OidcConfig {
     @ConfigItem(defaultValue = "false")
     public boolean resolveTenantsWithIssuer;
 
+    @ConfigItem
+    public DynamicTenants dynamicTenants = new DynamicTenants();
+
+    @ConfigGroup
+    public static class DynamicTenants {
+        /**
+         * If configured, limit the number of active OID tenant contexts to this value.
+         */
+        @ConfigItem
+        public int limit;
```

Comment:
I don't see this property used anywhere, yet, it is part of the config now.

Reply:
Yea - removed the config change